src/CMakeLists.txt

```diff
@@ -26,27 +26,50 @@ if (CMAKE_JS_VERSION)
 
     target_include_directories(${TARGET_NAME} PRIVATE ${CMAKE_JS_INC})
     target_link_libraries(${TARGET_NAME} PRIVATE ${CMAKE_JS_LIB})
-    
+
     # Using the V8 functions exported from node.exe
     target_compile_definitions(${TARGET_NAME} PRIVATE USING_V8_SHARED)
 else()
     # Building Napa for embed scenarios (static linking with v8)
 
-    if (NOT DEFINED V8_INC)
-        message(FATAL_ERROR "V8_INC must be set to the v8 include directory")
+    if (NOT DEFINED NODE_ROOT)
+        message(FATAL_ERROR "NODE_ROOT must be set to the node sources root directory")
     endif()
-    
-    if (NOT DEFINED V8_LIB_PATH)
-        message(FATAL_ERROR "V8_LIB_PATH must be set to the v8 libraries directory")
+
+    set(NODE_BUILD_TYPE "Release")
+    if ((DEFINED CMAKE_BUILD_TYPE) AND (CMAKE_BUILD_TYPE STREQUAL "debug"))
```

Comment:
>debug [](start = 67, length = 5)

differenct case of the 1st letter for 'debug' and 'Release'?

Reply:
This parameter is specified by the build.js script. It will always pass in a lowercase value.
I am transforming the first letter to capital to support node build structure in case sensitive operating systems like linux.

---
In reply to: [134074085](https://github.com/Microsoft/napajs/pull/47#discussion_r134074085) [](ancestors = 134074085)